resources/cluster-users/README.md

```diff
@@ -0,0 +1,3 @@
+# Cluster Users
+
+The `cluster-users` chart uses role-based access control (RBAC) to define roles and accesses in Kyma.
```

Comment:
```suggestion
The `cluster-users` chart uses role-based access control (RBAC) to define roles and access in Kyma.
```
`access` is usually uncountable. Can you switch it for `access`, `access control`, `or access permissions`?